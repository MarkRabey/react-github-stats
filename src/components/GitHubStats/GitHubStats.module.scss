$border-color: red;

.container {
  border: 1px solid $border-color;
}
