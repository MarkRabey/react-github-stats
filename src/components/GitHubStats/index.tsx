import * as React from 'react';

import styles from './GitHubStats.module.scss';
export interface GitHubStatsProps {
  title?: string;
}

const GitHubStats: React.FC<GitHubStatsProps> = ({title}) => {
  return (
    <div className={styles.container}>
      {title && (
        <header>
          <h1>GitHubStats</h1>
        </header>
      )}

      <p>This is GitHubStats</p>
    </div>
  );
};

export default GitHubStats;
