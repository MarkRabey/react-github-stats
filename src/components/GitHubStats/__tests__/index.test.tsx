import React from 'react';
import renderer from 'react-test-renderer';

import GitHubStats from '..';

describe('GitHubStats', () => {
  it('renders correctly', () => {
    const result = renderer.create(<GitHubStats />).toJSON();
    expect(result).toMatchSnapshot();
  });
});
