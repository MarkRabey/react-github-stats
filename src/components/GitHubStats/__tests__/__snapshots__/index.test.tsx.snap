// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubStats renders correctly 1`] = `
<div>
  <p>
    This is GitHubStats
  </p>
</div>
`;
